src/compiler/cpp_generator.cc

```diff
@@ -111,7 +111,8 @@ bool HasBidiStreaming(const grpc::protobuf::FileDescriptor *file) {
 }
 }  // namespace
 
-grpc::string GetHeaderIncludes(const grpc::protobuf::FileDescriptor *file) {
+grpc::string GetHeaderIncludes(const grpc::protobuf::FileDescriptor *file,
+                               const Parameters &params) {
```

Comment:
So, I usually like making API changes to be consistent - here I wanted something that can hold command line options, so that if someone else adds more options later on, they don't have to go back and modify the API. In fact, I might very well use this in the near future for another option. But if that's a problem, I'm not married to it.